mobify/platform-scaffold
# [Integration manager] merlins confirmation page

This PR moves the merlins confirmation page actions to the merlins connector and implements some missing functionality for updating a users shipping and billing info that was previously broken on desktop. It also moves the container fetchers into the router, this is because the app container needs the current container's fetcher function to provide offline support (modal reload button) and the react-loader component doesn't let us access the template component. 

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-50
 **Linked PRs**: n/a

## Changes
- Moved fetch confirmation page command to the merlins connector
- Moved checkout registration command to the merlins connector
- Fixed bug with shipping page for logged in user that prevented submitting the form
- Moved updateShippingAndBilling command to the merlins connector
- Fixed bug where new users addresses weren't actually being saved
- Added back commented out code since desktop is no longer broken
- Update confirmation page action tests
- Moved page fetchers to the router so offline support reload function works

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- add an item to your cart and complete an order
- the order confirmation page should work as expected
- *Note* this page should be tested as a logged in user and as a guest user
